.github/workflows/ci.yml

```diff
@@ -48,7 +49,20 @@ jobs:
           python-version: ${{ matrix.python }}
 
       - name: Run dbt-trino tests against ${{ matrix.engine }} on python ${{ matrix.python }}
-        run: make dbt-${{ matrix.engine }}-tests
+        env:
```

Comment:
This step should be executed only when the secrets are present (basically when doing a branch from `starburstdata/dbt-trino` repository (and not from one of its clones).

See Trino's approach for this use case: https://github.com/trinodb/trino/blob/master/.github/workflows/ci.yml#L533-L547

Reply:
How we can test then PRs from forks? How Trino approach handles that?

Reply:
Generally in such cases a maintainer does a copy of the pr (once that the maintainer ensures that the changes do not contain code which intentionally prints out to the console the credentials) in the trino repository and run the PR build with secrets.

Reply:
It sounds like not ideal solution. I wonder how other projects handle it, like for example dbt-snowflake https://github.com/dbt-labs/dbt-snowflake/blob/main/.github/workflows/integration.yml#L160-L172

Reply:
Actually I don't think it's an issue here, because we still need to approve workflow run from forks like described here https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks#about-workflow-runs-from-public-forks